MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>bad.</p><p> </p><pre class="ql-syntax" spellcheck="false"># good practice: immutable
df = df.drop(columns="CODGEO")
</pre><p> </p><pre class="ql-syntax" spellcheck="false"># not so good practice but can be useful on BIG dataframes (I mean VERY BIG)
df.drop(columns="CODEGEO", inplace=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>